app/controllers/admin/trends/tags_controller.rb

```diff
@@ -22,7 +22,7 @@ def batch
   private
 
   def filtered_tags
-    Trends::TagFilter.new(filter_params).results
+    Trends::TagFilter.new(filter_params).results.includes(:user)
```

Comment:
Something is wrong here too, `Tag` doesn't have a user association, and I don't see anything using this in the template.